clients/buzz-adapter.rst

```diff
@@ -54,6 +54,13 @@ Or if you installed the :doc:`discovery </discovery>` layer::
 
     The message factory parameter is mandatory if the discovery layer is not installed.
 
+Be aware
+--------
+
+This adapter violates the Liskov substitution principle in a rare edge case. When the adapter is configured to use
+Buzz' Curl client, it does not send request bodies for request methods such as GET, HEAD and TRACE. A RequestException
+will be thrown if this ever happens.
```

Comment:
lets add: "The only solution if you need GET requests with a body (e.g. for Elasticsearch) is to use <whatever> in Buzz or a different client like Guzzle 6.